libs/ledger/include/ledger/transaction_status_cache.hpp

```diff
@@ -18,24 +18,18 @@
 //------------------------------------------------------------------------------
 
 #include "core/digest.hpp"
-#include "core/mutex.hpp"
-#include "core/synchronisation/waitable.hpp"
 #include "ledger/execution_result.hpp"
-#include "network/generics/milli_timer.hpp"
-
-#include <chrono>
-#include <unordered_map>
 
 namespace fetch {
 namespace ledger {
 
 enum class TransactionStatus : uint8_t
 {
-  UNKNOWN,    ///< The status of the transaction is unknown
-  PENDING,    ///< The transaction is waiting to be mined
-  MINED,      ///< The transaction has been mined
-  EXECUTED,   ///< The transaction has been executed
-  SUBMITTED,  ///< Special case for the data based synergetic transactions
+  UNKNOWN = 0,  ///< The status of the transaction is unknown
```

Comment:
Why the explicit 0?

Reply:
No real need, just think that it is clearer